spec/unit/util/windows/adsi_spec.rb

```diff
@@ -144,6 +163,14 @@
       Puppet::Util::Windows::ADSI::User.delete(username)
     end
 
+    [:STANDALONE_WORKSTATION, :MEMBER_WORKSTATION, :STANDALONE_SERVER, :MEMBER_SERVER, :BACKUP_DOMAIN_CONTROLLER, :PRIMARY_DOMAIN_CONTROLLER].each_with_index do |role, index|
```

Comment:
Instead of redoing the array here, should it be using the constants from `Puppet::Util::Windows::ADSI` ?